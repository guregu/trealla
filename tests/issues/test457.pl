:- initialization(main).

main :-
	length(L,4), append(L,[_|Z],LT), [X,Y,Z] = [x,[X,_|Z],[_,y|Z]], Y = LT , Z = LT, [_,_,_,_,X3,Y3|_] = LT.
