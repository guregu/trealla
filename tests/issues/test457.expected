Warning: Initialization goal failed: main
